GpsDataCapturer/sharedlibrary/src/main/java/com/google/sharedlibrary/service/GpsDataCaptureService.java

```diff
@@ -177,10 +167,8 @@ public void onLowMemory() {
 
     /**
      * Start capturing data from GPS via the chosen location api
-     *
-     * @param locationApiType the chosen location api
      */
-    public void startCapture(LocationApiType locationApiType) {
```

Comment:
why do you plan to remove this?

Reply:
The change is I make the `locationApiType` a private field in the service, and call `setLocationApiType()` it in the main activity. Then we don't need to pass`locationApiType` as a parameter to `stopCapture()`. Therefore to stop capture via command line, we don't need to extras. ```adb shell am startservice -a com.google.gpsdatacapturer.STOP_CAPTURE ``` is sufficient.

Reply:
ack